featuretools/primitives/standard/cum_transform_feature.py

```diff
@@ -47,8 +47,9 @@ class CumCount(TransformPrimitive):
         [1, 2, 3, 4, 5, 6]
     """
     name = "cum_count"
-    input_types = [[Id], [Discrete]]
-    return_type = Numeric
+    input_types = [[ColumnSchema(semantic_tags={'foreign_key', 'category'})],
```

Comment:
Yeah, I think the fact that `Id` variable type has `_default_pandas_dtype = int` but is a subclass of Categorical was one of those things we won't exactly be able to recreate between variables and Woodwork. Agree that we probably just want the `foreign_key` tag here, because it's not uncommon for the columns to have `Integer` logical types.